src/Libraries/Microsoft.Extensions.Diagnostics.ExtraAbstractions/Http/TelemetryConstants.cs

```diff
@@ -11,7 +11,7 @@ public static class TelemetryConstants
     /// <summary>
     /// Request metadata key that is used when storing request metadata object.
     /// </summary>
-    public const string RequestMetadataKey = "R9-RequestMetadata";
+    public const string RequestMetadataKey = "Extensions-RequestMetadata";
```

Comment:
I'm assuming we do want to make this change, yet we do expect this to cause breaks for teams currently depending on this. @geeknoid can you confirm?

Reply:
This should be fine, it's something teams will have to deal with when upgrading.